ppb/engine.py

```diff
@@ -8,6 +8,19 @@
 class GameEngine(Engine):
 
     def __init__(self, first_scene: Type, **kwargs):
+        """
+        Game Engine
+
+        first_scene -- the scene to be activated first
+
+        delta_time -- time between frames
+
+        resolution -- window (w, h) tuple
+
+        log_level -- log level for all messages
```

Comment:
log level is the minimum log level for printing to the console. It defaults to logging.WARNING to minimize end user noise, but the keyword exists specifically for developers to switch down to INFO or DEBUG for debugging.